Project/Assets/ML-Agents/Examples/Basic/Scripts/BasicAgent.cs

```diff
@@ -92,16 +85,21 @@ public void FixedUpdate()
 
     void WaitTimeInference()
     {
-        if (!Academy.Instance.IsCommunicatorOn)
+        if (Academy.Instance.IsCommunicatorOn)
```

Comment:
The logic got flipped here accidentally at some point. Now training steps quickly and inference takes its time.

Reply:
😨